### PR TITLE
Update README with usage instructions

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +22,101 @@ The following platforms are officially supported (tested):
...
+)
+```
+
+### Streaming Public Trades
+
+To get real-time updates on market trades, one can use the following code snippet.
+
+```python
+async for order_detail in client.stream_public_trades():
+    print(f"Received public trade trade: {order_detail}")
```
Comment (daniel-zullo-frequenz) [inline README.md]:
```suggestion
async for public_trade in client.stream_public_trades():
    print(f"Received public trade: {public_trade}")
```

File: README.md [hunk 2]
```diff
@@ -16,6 +22,101 @@ The following platforms are officially supported (tested):
...
+```
+
+### Initialization
+
+First, initialize the client with the appropriate server URL and API key.
+```python
+from frequenz.client.electricity_trading import Client
+
+# Change server address if needed
+SERVICE_URL = "electricity-trading-testing.api.frequenz.com:443"
```
Comment (daniel-zullo-frequenz) [inline README.md]: we will need to set the URL to use the latest version of the client as follows:
```suggestion
SERVICE_URL = "grpc://electricity-trading-testing.api.frequenz.com:443?ssl=true"
```

File: README.md [hunk 3]
```diff
@@ -16,6 +22,101 @@ The following platforms are officially supported (tested):
...
+API_KEY = open('api_key.txt').read().strip()
```
Comment (daniel-zullo-frequenz) [inline README.md]: Maybe clarify a path might be needed, for instance, `'/path/to/api_key.txt'`

**Review by daniel-zullo-frequenz — COMMENTED** [review]
I have a few suggestions to check for, LGTM otherwise

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Would it make sense to have it in 2 steps like so or rather not?
```
SERVICE_ADDRESS = "electricity-trading.api.frequenz.com:443"
SERVICE_URL = f"grpc://{SERVICE_ADDRESS}?ssl=true"
```

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: uups good call

File: README.md (hunk 2 above)
Comment (daniel-zullo-frequenz) [reply README.md]: I personally prefer 1 step/line as the address will be only used in the URL. Though I don't have any strong opinion about doing it in 2 steps

**Review by daniel-zullo-frequenz — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: makes sense, I just thought that in the future the `SERVICE_ADDRESS` might be in the config in the app, while `SERVICE_URL` is more implementation specific but actually I guess we can just have the URL in the config

**Review by daniel-zullo-frequenz — APPROVED** [review]

**Pull request merged**